setupBenchmark.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -exo pipefail
+
+# Contains OS specific sed function
+. benchmarks/setup/utils.sh
+
+if [ -z $1 ]
+then
+  echo "Please provide a benchmark name! Format should be 'YOUR_NAME-TOPIC'"
+  exit 1
+fi
+benchmark=$1
+
+mvn clean install -DskipTests -T1C
+
+docker build --build-arg DISTBALL=dist/target/zeebe-distribution-*.tar.gz --build-arg APP_ENV=dev -t "gcr.io/zeebe-io/zeebe:$benchmark" .
```

Comment:
What does `APP_ENV=dev` achieve?

Reply:
Uses a jdk image and installes profiler to the image. Makes it easier to debug and profile the benchmark

https://github.com/zeebe-io/zeebe/blob/develop/Dockerfile#L28-L31

I normally use this image.

Reply:
I which i would've known this sooner

Reply:
I added this a while ago because it annoyed me as well quite often :laughing: I think I have mentioned in slack :sweat_smile:

Reply:
I must have missed it, thanks though. Great addition